packages/api/api.ts

```diff
@@ -3324,6 +3517,310 @@ export interface NftPositionWithMarketValues {
   portfolioPercentage?: number | null
 }
 
+export interface PortfolioBrokerBaseRequest {
+  /**
+   * Auth token that allows connecting to the target institution
+   * @minLength 1
+   */
+  authToken: string
+  /**
+   * Type of the institution to connect
+   *
+   * ### Supported integrations:
+   * ```Robinhood```
+   * ```ETrade```
+   * ```Alpaca```
+   * ```TdAmeritrade```
+   * ```WeBull```
+   * ```Stash```
+   * ```InteractiveBrokers```
+   * ```Public```
+   * ```Coinbase```
+   * ```Kraken```
+   * ```CoinbasePro```
+   * ```CryptoCom```
+   * ```OpenSea```
+   * ```Binance```
+   * ```Gemini```
+   * ```OkCoin```
+   * ```Bittrex```
+   * ```KuCoin```
+   * ```Etoro```
+   * ```CexIo```
+   * ```BinanceInternational```
+   * ```Bitstamp```
+   * ```GateIo```
+   * ```Acorns```
+   * ```Okx```
+   * ```BitFlyer```
+   * ```Coinlist```
+   * ```Huobi```
+   * ```Bitfinex```
+   * ```KrakenDirect```
+   * ```Vanguard```
+   * ```BinanceInternationalDirect```
+   * ```BitfinexDirect```
+   * ```Bybit```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
+   * ```DeFiWallet```
```

Comment:
is it ok?

Reply:
its not, i guess this is BE error

Reply:
it's not a blocker for the release, right? @salavatgainetdinov-front @balint-front